sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -198,11 +198,14 @@ class SourceProgress protected[sql](
  * during a trigger. See [[StreamingQueryProgress]] for more information.
  *
  * @param description Description of the source corresponding to this status.
+ * @param numOutputRows Number of rows written to the sink or -1 for Continuous Mode (temporarily)
+ * or Sink V1 (until decommissioned).
  * @since 2.1.0
  */
 @InterfaceStability.Evolving
 class SinkProgress protected[sql](
-    val description: String) extends Serializable {
+  val description: String,
+  val numOutputRows: Long) extends Serializable {
```

Comment:
I feel `numOutputRows` is a bit confusing. It sounds like how many rows outputted by the Sink. But the real meaning is how many rows written to the sink. How about `numReceivedRows`?

Reply:
numOutputRows is used here to be similar to SourceProgress#numInputRows. I agree that I could change from SinkProgress#numOutputRows to SinkProgress#numInputRows as it may make bit more sense from what actually happens. Would you and others be in favor of that?

Reply:
@zsxwing, what do you think?